crates/nu-cli/src/repl.rs

```diff
@@ -307,6 +308,13 @@ pub fn evaluate_repl(
             column!()
         );
 
+        if entry_num == 1 && show_banner {
```

Comment:
Not sure I understand this `entry_num == 1` check. What's its purpose?

Reply:
The point of `entry_num == 1` is to ensure this output only happens at startup. Without that, you'll get an ever increasing output with every single command.